modules/fedimint-mint/src/lib.rs

```diff
@@ -455,16 +452,21 @@ impl std::fmt::Display for MintConsensusItem {
     }
 }
 
+pub struct MintModuleTypes;
```

Comment:
i was wondering if I could come up with a better name that does not include `types` as it's a bit generic but I couldn't. And perhaps generic/or general is what we want here, as this represents the common types/structs/required consensus types shared between the client and server part of a module. Also thinking with regards to module implementors what `ModuleCommon` is and the trait documentation says that, so :+1: